node/orchestra/proc-macro/Cargo.toml

```diff
@@ -28,10 +29,10 @@ thiserror = "1"
 tracing = "0.1"
 
 [features]
-default = []
+default = ["graph"]
```

Comment:
what was the reason to disable it by default? I'm not sure why need it by default

Reply:
The reason was some issue with the CI system on `substrate` https://github.com/paritytech/ci_cd/issues/433 - I did not dig into this just yet.